composer.json

```
@@ -19,7 +19,7 @@
   "require": {
     "php": "^7.1",
```

Monolog 2 [requires at least PHP 7.2](https://github.com/Seldaek/monolog/blob/master/UPGRADE.md), so:
```suggestion
     "php": "^7.2",
```